nekoyume/Assets/_Scripts/Extension/DateTimeExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace Nekoyume
+{
+    public static class DateTimeExtensions
+    {
+        public static bool IsInTime(this DateTime value, DateTime begin, DateTime end)
+        {
+            var bDiff = (value - begin).TotalSeconds;
+            var eDiff = (end - value).TotalSeconds;
+            return bDiff > 0 && eDiff > 0;
+        }
+
+        /// <param name="value"></param>
+        /// <param name="begin">"yyyy-MM-ddTHH-mm-ss"</param>
+        /// <param name="end">"yyyy-MM-ddTHH-mm-ss"</param>
+        /// <returns></returns>
+        public static bool IsInTime(this DateTime value, string begin, string end) =>
+            value.IsInTime(
+                DateTime.ParseExact(begin, "yyyy-MM-ddTHH:mm:ss", null),
```

Comment:
The dateTime format of the comment and the dateTime format of the code appear to be different. Is it okay?
comment : `"yyyy-MM-ddTHH-mm-ss"`
code : `"yyyy-MM-ddTHH:mm:ss"`